lib/kitchen/directions/bake_further_research.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    # Bake directions for further research
+    #
+    module BakeFurtherResearch
+      def self.v1(chapter:, metadata_source:)
+        metadata_elements = metadata_source.children_to_keep.copy
+
+        further_researches = Clipboard.new
+
+        # TODO: include specific page types somehow without writing it out
```

Comment:
this comment can be removed since `non_introduction_pages` has been implemented!

Reply:
@kswanson33 I missed it. Thank you!